protocols/imap/src/test/java/org/apache/james/imap/processor/base/FakeMailboxListenerAdded.java

```diff
@@ -33,12 +33,14 @@
 import org.apache.james.mailbox.model.MessageId;
```

Comment:
Remove that useless class please.

Use directly `MailboxListener.Added` in `MailboxEventAnalyserTest`